livekit_analytics.proto

```diff
@@ -19,9 +19,16 @@ enum StreamType {
   DOWNSTREAM = 1;
 }
 
-message AnalyticsStream {
+message AnalyticsVideoLayer {
```

Comment:
Taking the opportunity to clean up and making it possible to use it for both simulcast/non-simulcast/SVC streams.

Limiting to spatial layers only. Not doing temporal/quality layer granularity as it could get too complicated.

So, I have split things as follows
- `AnalyticsStat` -> Track level info. This has an array of `AnalyticsStream`. One per SSRC in that track. For non-simulcast/SVC tracks, there will be only one element in that list. For simulcast streams, this will have as many streams as the simulcast stream is using.
- `AnalyticsStream` -> RTP Stream (identified by SSRC) level information. This has an array of `ArrayVideoLayer`. For audio, that will be empty. For non-simulcast video also, it will be empty (or it can have one layer if that is preferable). For simulcast or SVC, the array will have as many spatial layers in the stream.
- `AnalyticsVideoLayer` -> Per spatial layer information. Stats that are applicable here are `total bytes`, `total packets` and `total frames`.